pom.xml

```diff
@@ -116,7 +116,9 @@
 
     <processorVersion>1.3</processorVersion>
 
-    <doxiaVersion>1.6</doxiaVersion>
+    <doxiaVersion>1.8</doxiaVersion>
+    <!-- ASF project maven-doxia-sitetools:1.8.1 has different release cycle and it is dependent on maven-doxia:1.8 -->
```

Comment:
@ppalaga here is the comment

Reply:
We can see the Doxia projects on GitHub, namely doxia and renderer:
https://github.com/apache/maven-doxia
https://github.com/apache/maven-doxia-sitetools